src/QUESTIONS.md

```diff
@@ -46,6 +54,28 @@ website and follow the instructions there.
    * For CHERI-RISC-V: `cheribuild.py run-riscv64-purecap -d`
 
 
+### Where are the cheribuild.py images and builds stored?
+
+By default, any images will reside in `$HOME/cheri/output` and builds in `$HOME/cheri/build`.
+
+
+### Can I build targets from specific branches using cheribuild.py?
+
+Yes. Once executed, the cheribuild.py script will create a `$HOME/cheri` directory to contain all of the repositories, builds, and configuration needed. If you change to the specific directory for your target, you will then be able to check out the correct branch and run `./cheribuild.py` against it.
+```bash
+cd $HOME/cheri/cheribsd
+git checkout releng/22.12
+
+cd $HOME/cheribuild
+./cheribuild.py cheribsd-morello-purecap
+```
+
+
+### Can I develop bare-metal applications for Morello?
+
+Yes. Arm have created an example of a [bare-metal application](https://git.morello-project.org/morello/docs/-/blob/morello/mainline/common/standalone-baremetal-readme.rst) for the board.
```

Comment:
I've no preference. It was among the participants' questions from a previous cohort and happened to be orphaned as the only question of its kind.

Reply:
I think it would be useful to have a reference to a document describing how to run a baremetal application.

We want to include information on software stacks developed outside Cambridge/SRI. There are people (including members of TAP) working on seL4 and FreeRTOS. There were also questions regarding baremetal applications at CHERI-CPU.